### PR TITLE
Fixes #31344 - Update @theforeman/builder

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@sheerun/mutationobserver-shim": "^0.3.3",
     "@testing-library/jest-dom": "^5.3.0",
     "@testing-library/react": "^10.0.2",
-    "@theforeman/builder": "^4.6.0",
+    "@theforeman/builder": "^6.0.0",
```
Comment (tbrisker) [inline package.json:29]: Should this be `>= 6.0.0` to ensure it doesn't break on the next update?

Comment (johnpmitsch) [reply package.json:29]: we could change to that for builder for now. Ideally we don't have to update all the foreman-js metapackages at the same time so a major version change in `@theforeman/builder` did mean a breaking change. I'm opening up a discussion on this topic.

Comment (johnpmitsch) [reply package.json]: @tbrisker updated to use `>=`

**Comment by evgeni** [comment]
[test katello]

**Comment by evgeni** [comment]
The three failing tests are unrelated (Pulp3 VCR tapes don't match) and also fail on other PRs.

**Review by tbrisker — APPROVED** [review]
Thanks @johnpmitsch !

**Pull request merged**